packages/components/src/components/List/UnorderedList.tsx

```diff
@@ -0,0 +1,49 @@
+import styled from "@xstyled/styled-components";
+import React, { ReactElement } from "react";
+import { Box } from "../../primitives/Box";
+import { ListItemProps } from "./ListItem";
+
+export type ListFontColors = "colorText" | "colorTextLink";
+export interface UnorderedListProps
+  extends Omit<React.HTMLAttributes<HTMLUListElement>, "color"> {
+  /** Changes the color of the list items. */
+  color?: ListFontColors;
```

Comment:
Should this type be part of the ListItem? 🤔

Reply:
Why does a list need a color?

Reply:
When we have an anchor list the bullet points should have a different color. But I just realized that it can follow the container color. So, I updated the stories.